cmds/webboot/webboot.go

```diff
@@ -97,7 +147,7 @@ func (d *DownloadOption) exec(uiEvents <-chan ui.Event, network bool) (menu.Entr
 
 // DirOption's exec displays subdirectory or cached isos under the path directory
 func (d *DirOption) exec(uiEvents <-chan ui.Event) (menu.Entry, error) {
-	entries := []menu.Entry{}
+	entries := []menu.Entry{&BackOption{}}
```

Comment:
This is just a personal preference but what do you think about having the back option be the last thing in the menu, not the first?

Reply:
ok, I will change that